canonical-web-and-design/ubuntu.com
# Build cve main page

## Done

Add CVE table to main CVE page

## QA

- Check out this feature branch
- `docker-compose up -d`
- `dotrun --env DATABASE_URL=postgres://postgres:pw@localhost:5432/postgres`
- `dotrun exec python3 webapp/security/fixtures/load_sample.py`
- `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/cve
- Check that there is a table showing a list of CVEs with packages and the releases they affect as per [Matthew's sketch](https://docs.google.com/document/d/1VuAElUpFay7MBFNmE3NhSyY9TkXiqHcA1ZHQ11AYO-Q/edit?disco=AAAAGZDIyRw)

**Code:** Please only review the changes to `templates/security/cve.html` and `templates/security/cve/index.html`. All other changes are taken from #7173 which this branch is based on.

**Note:** There is no search or pagination as that will be addressed in #7056 

## Issue / Card

Fixes #7059 

## Screenshots

![Screenshot from 2020-04-09 13-53-31](https://user-images.githubusercontent.com/501889/78897120-9947dd80-7a69-11ea-94e4-ccb975fd6fe3.png)

![screencapture-localhost-8001-security-cve-1586436760185](https://user-images.githubusercontent.com/501889/78897144-a2d14580-7a69-11ea-89bd-78ae85784098.png)
